tests/test_tracing.py

```diff
@@ -340,9 +354,12 @@ def test_with_sqs_distributed_datadog_trace_data(self):
             },
         )
         create_dd_dummy_metadata_subsegment(ctx, XraySubsegment.TRACE_KEY)
+        args, kwargs = self.mock_send_segment.call_args
+        self.assertEqual(args[0], XraySubsegment.TRACE_KEY)
+        actual_context = args[1]
         self.mock_send_segment.assert_called_with(
             XraySubsegment.TRACE_KEY,
-            {"trace-id": "123", "parent-id": "321", "sampling-priority": "1"},
+            actual_context
```

Comment:
This is wrong. You should be doing assertions against the `args` returned from `.call_args`.